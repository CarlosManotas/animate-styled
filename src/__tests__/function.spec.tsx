import * as React from 'react';
import AnimateStyled from '..';
import * as renderer from 'react-test-renderer';

test("Component should show 'red' text 'Hello World'", () => {
  const component = renderer.create(<AnimateStyled name="backInDown" />);
  const testInstance = component.root;

  expect(testInstance.findByType(AnimateStyled).props.name).toBe('backInDown');

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
