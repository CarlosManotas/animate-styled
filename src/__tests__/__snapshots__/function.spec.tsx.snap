// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show 'red' text 'Hello World' 1`] = `
<div
  className="sc-bdfBwQ fjuHEr"
  direction="normal"
  name="bounceOut"
  opacity={1}
>
  Hello World
</div>
`;
